include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -99,7 +99,7 @@ MEMORY
     SRAM                  (wx) : ORIGIN = RAM_ADDR, LENGTH = RAM_SIZE
 
     /* Used by and documented in include/linker/intlist.ld */
-    IDT_LIST  (wx)      : ORIGIN = (RAM_ADDR + RAM_SIZE), LENGTH = 2K
+    IDT_LIST  (wx)      : ORIGIN = (RAM_ADDR + RAM_SIZE), LENGTH = 10K
```

Comment:
Why is is ths 10K, now? Was this an arbitrary selection?

Reply:
this value is arbitrary. the data here is interrupt routing information which gets stripped out of the binary and used by the build scripts. it is placed in an alternate memory are so this data doesn't count against total available ram size. this change is fine.